pages/index.tsx

```diff
@@ -85,6 +86,35 @@ const AssetCardTitle = styled(Card.HeaderTitle)`
   overflow: hidden;
 `
 
+const Hero = styled.section`
+  display: grid;
+  grid-template-areas: "hero";
+  min-height: 28rem;
+  align-items: center;
+  padding: 2rem 0;
+   > * {
+    grid-area: hero;
+  }
+`
+
+const HeroContent = styled.div`
+  width: clamp(25ch, 60%, 600px);
+  z-index: 1;
+  align-self: start; 
+  background-color: rgba(255, 255, 255, 0.85);
+  border-radius: ${tokens.shape.corners.borderRadius};
+  padding: 0.5rem 0.5rem 0.5rem 0;
```

Comment:
smort! maybe the background-color should be solid white? and i agree (😏), maybe 0.5rem is a bit too sparse. we could probably increase it to 1 or even 1.5 rem

Reply:
Dunno? It's a matter of taste I guess! But the contrast should be good enough with a semitransparent background as I double-checked that. Let's try 1rem

Reply:
Nah, didn't look too good with 1rem at least with the current text content. We could fine tune when we have the final text 👍  EDS does support mod 4 spacings, so it's not technically wrong